pydantic/schema.py

```diff
@@ -274,6 +282,7 @@ def field_schema(
         model_name_map=model_name_map,
         schema_overrides=schema_overrides,
         ref_prefix=ref_prefix,
+        known_models=known_models,
```

Comment:
```suggestion
        known_models=known_models or set(),
```

and the line above can be removed.

Reply:
Wouldn't this possibly lead to issues with "Mutable Default Arguments"? https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
no, this is calling the function not defining it I think.

Reply:
Ah! Right. I thought this comment was in the definition.